scripts/ssb-patchfoo/install

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+# shellcheck disable=SC1091
+true
+
+BASE_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+current="$(pwd)"
+
+# Install nginx
+# shellcheck source=../shared/nginx/install
+source "$BASE_DIR/../shared/nginx/install"
+
+sleep 5
+# shellcheck disable=SC2164
+cd ~/.ssb/node_modules
+
+# Install dependencies
+npm install asyncmemo hashlru pull-stream pull-cat multicb hyperscript pull-paramap ssb-contact ssb-sort stream-to-pull-stream emoji-server pull-paginate ssb-mentions busboy mime-types pull-identify-filetype human-time pull-hyperscript jpeg-autorotate pull-catch diff pull-split pull-utf8-decoder ssb-web-resolver highlight.js pull-box-stream base64-url ssb-backlinks ssb-private
+
+# Install patchfoo and enable plugin
+git clone https://github.com/ssbc/patchfoo.git patchfoo
+sbot plugins.install ssb-private
+sbot plugins.install ssb-backlinks
+sbot plugins.enable patchfoo
+
+# Stop ssb service to process plugin
+sudo systemctl stop ssb
```

Comment:
Disable and reenable to be safe?

Reply:
The requirement here is a restart of the service. I stop it for good measure.

Even if you don't restart it the script will reboot the pi in the end.